server/chat-plugins/thing-of-the-day.ts

```diff
@@ -230,7 +230,7 @@ class OtdHandler {
 		const entries = [];
 
 		for (const value of this.nominations.values()) {
-			entries.push(`<li><b>${value.nomination}</b> <i>(Submitted by ${value.name})</i></li>`);
+			entries.push(`<li><b>${value.nomination}</b> <i>(Submitted by ${Utils.escapeHTML(value.name)})</i></li>`);
```

Comment:
```suggestion
			entries.push(Utils.html`<li><b>${value.nomination}</b> <i>(Submitted by ${value.name})</i></li>`);
```
IIRC we prefer this style.